### PR TITLE
Release v0.12.1-alpha.1

### PR DESCRIPTION
**Pull request by roivaz**
This PR migrates the marin3r upstream catalog to the [file based catalog format](https://olm.operatorframework.io/docs/reference/file-based-catalogs/). A couple of important things:

* I have cleaned up all alpha bundles from the catalog (there were a lot of them) and added the `skipRanges` option to the alpha catalog so any alpha bundle will be directly upgraded to its corresponding stable bundle. For example `marin3r.v0.11.0-alpha.5` will be upgraded directly to `marin3r.v0.11.0` instead of installing all the alpha releases in between.

* I have included v0.12.0 in the skipped range for v0.11.1 so it's never installed, as it contains the now infamous upgrade bug. I added a comment in the alpha channel to clarify this.

This PR just updates the alpha channel, I'll test v0.12.1-alpha.1 in staging and then upgrade the stable channel and re-test the upgrade path in staging.

/kind feature
/kind bug
/priority important-soon
/assign

### CODE REVIEW

File: examples/e2e/deployment/v0.12.0/envoydeployment.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        - type: Resource
+          resource:
+            name: cpu
+            target:
+              type: Utilization
+              averageUtilization: 50
+  podDisruptionBudget:
+    minAvailable: 1
+  shutdownManager: {}
+    # image: quay.io/3scale/marin3r:test
```
Comment (slopezz) [inline examples/e2e/deployment/v0.12.0/envoydeployment.yaml:36]: are needed those comments?

**Review by raelga — APPROVED** [review]

**Review by raelga — APPROVED** [review]

**Review by raelga — APPROVED** [review]

File: examples/e2e/deployment/v0.12.0/envoydeployment.yaml [hunk 2]
```diff
@@ -34,5 +34,5 @@ spec:
     minAvailable: 1
   shutdownManager:
     image: quay.io/3scale/marin3r:test
-  initManager:
-    image: quay.io/3scale/marin3r:test
\ No newline at end of file
+  # initManager:
+  #   image: quay.io/3scale/marin3r:test
```
Comment (slopezz) [inline examples/e2e/deployment/v0.12.0/envoydeployment.yaml:38]: comments??

**Pull request merged**